polling_stations/apps/data_importers/management/commands/import_wirral.py

```diff
@@ -5,54 +5,76 @@
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "WRL"
     addresses_name = (
-        "2023-05-04/2023-02-22T10:54:17.531032/Democracy_Club__04May2023.tsv"
+        "2024-05-02/2024-02-19T13:46:05.384042/Democracy_Club__02May2024 Wirral.tsv"
     )
     stations_name = (
-        "2023-05-04/2023-02-22T10:54:17.531032/Democracy_Club__04May2023.tsv"
+        "2024-05-02/2024-02-19T13:46:05.384042/Democracy_Club__02May2024 Wirral.tsv"
     )
-    elections = ["2023-05-04"]
+    elections = ["2024-05-02"]
     csv_delimiter = "\t"
 
     def station_record_to_dict(self, record):
-        # Marlowe Road URC Hall
-        if record.polling_place_id in ["10084", "10088"]:
+        # Marlowe Road URC Hall (Station A), Marlowe Road, Wallasey, Wirral, CH44 3DG
```

Comment:
These ids are for `Station A` and `Station B`, so I think the comment made more sense before.